.storybook/webpack.config.js

```diff
@@ -25,6 +31,13 @@ module.exports = async ( { config } ) => {
 	config.plugins = [
 		...config.plugins,
 		new MiniCssExtractPlugin(),
+		new FeatureFlagsPlugin(
+			flagsConfig,
+			{
+				modes: [ 'development', 'production' ],
+				mode: config.mode,
+			},
+		),
```

Comment:
Just to clarify, why is this needed in the Storybook Webpack?

Reply:
@felixarntz this was added to allow access to the featureFlags. I have removed it and instead am setting the flag in the story.